src/pages/open source for non-developers.md

```diff
@@ -0,0 +1,23 @@
+# Open Source for Non-Developers
+
+## What is Open Source?
+
+Open source is software containing source code that is freely modified and redistributed. Unlike products that contain proprietary code, open source code can be viewed by developers worldwide. It can also allows people to propose changes, which fosters a collaborative working environment.
```

Comment:
[Proofreading]
* I think this is a technical correction? Something isn't necessarily open source if it's only _partially_ consisting of open source code.
* Freely modifiable isn't right - just because someone suggests a modification, doesn't mean it's going to be accepted. (Wikipedia is not the model we want people to think of)

```suggestion
Open source is software whose source code that can be freely viewed and redistributed. Unlike products that contain proprietary code, open source code can be viewed by developers worldwide. It can also allows people to propose changes, which fosters a collaborative working environment.
```

(fun fact, almost all proprietary code contains some open source dependencies!)